playbooks/site.yml

```diff
@@ -19,3 +19,19 @@
     - role: containerd
       tags:
         - containerd
+
+- name: Install CRI tools
+  hosts: "{{ target | default('all') }}"
+  become: true
+  roles:
+    - role: cri_tools
+      tags:
+        - cri-tools
+
+- name: Install CNI plugin
+  hosts: "{{ target | default('all') }}"
+  become: true
+  roles:
+    - role: cni_plugins
+      tags:
+        - cni-plugins
```

Comment:
@ricolin shouldn't these technically be role dependencies instead?

Reply:
you mean role dependencies in vexxhost.kubernetes?